torch/_inductor/codegen/cuda/gemm_template.py

```diff
@@ -161,10 +184,12 @@ def __init__(
         alpha: float,
         beta: float,
         input_reorder: Optional[List[int]] = None,
+        can_fuse_epilogue: bool = False,
     ):
         super().__init__("cutlass_gemm", input_nodes, layout, input_reorder)
         self.alpha = alpha
         self.beta = beta
+        self.can_fuse_epilogue = can_fuse_epilogue
```

Comment:
If this flag is set, it will only list operations which support EVT. If it is false, it will only list operations which don't.